pkg/controller/enterprisesearch/version_upgrade_test.go

```diff
@@ -312,3 +342,43 @@ func TestVersionUpgrade_readOnlyModeRequest(t *testing.T) {
 		})
 	}
 }
+
+func TestVersionUpgrade_isVersionUpgrade1(t *testing.T) {
```

Comment:
The existing `TestVersionUpgrade_isVersionUpgrade` should be renamed `TestVersionUpgrade_isPriorVersionStillRunning`, this would allow to rename this one `TestVersionUpgrade_isVersionUpgrade`